afdd_stack/openfdd_stack/tests/platform/test_bootstrap_modes.py

```
@@ -81,3 +83,83 @@ def test_bootstrap_rejects_unknown_option():
+
+
+def test_volttron_duplicate_ssl_quarantine_respects_strict(tmp_path):
+    cfg = tmp_path / "config"
+    cfg.write_text(
+        "[volttron]\nweb-ssl-cert = /a.pem\nweb-ssl-cert = /b.pem\n",
+        encoding="utf-8",
+    )
+    script = "export OFDD_VOLTTRON_CONFIG_STRICT=1\n"
+    script += _duplicate_ssl_quarantine_function_source()
+    script += f'run_volttron_quarantine_duplicate_ssl_config "{cfg}"\n'
+    res = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)